src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -113,9 +113,8 @@ describe('ComputeModal', () => {
 
     // Assert
     verifyEnabled(getCreateButton())
-    const title = screen.getByText('Jupyter Cloud Environment')
-    expect(title).toBeInTheDocument()
-    expect(screen.getByText('Create custom environment')).toBeInTheDocument()
+    screen.getByText('Jupyter Cloud Environment')
```

Comment:
Consider a matcher like `toBeTruthy` to clarify that these are assertions instead of relying on this to throw an error if the text is not found.

```suggestion
    expect(screen.getByText('Jupyter Cloud Environment')).toBeTruthy()
```

Reply:
Also, consider using queries returned by `render` instead of queries on `screen`.

https://testing-library.com/docs/react-testing-library/api#queries-1

Reply:
So we had an expect before, Miguel recommends not asserting anything on getters as it makes the error message a bit clearer. Sounds like we need to start having a standards talk. I'm going to keep this as is since I have received conflicting opinions at this point.